### PR TITLE
Text-to-Speech-support-#53

### PR DESCRIPTION
**Pull request by hunght**
Sorry for take it so long, pls review this for me.
Best regards
HungHT

### CODE REVIEW

**Comment by hunght** [comment]
@hebertialmeida travis tell me there was a bug when build app, it's about no signature ...
could you check that for me

**Comment by hebertialmeida** [comment]
@hunght I will check that, but I am new to travis and It is not building the project correctly.

File: Source/FolioReaderAudioPlayer.swift [hunk 1]
```diff
@@ -61,23 +75,82 @@ class FolioReaderAudioPlayer: NSObject, AVAudioPlayerDelegate {
...
+                    utteranceRate = 0.15
+                    break
+                case 3:
+                    utteranceRate = 0.23
+                    break
+                default:
+                    break
+                }
+            }
+
```
Comment (hebertialmeida) [inline Source/FolioReaderAudioPlayer.swift:116]: You will need to call `updateNowPlayingInfo()` here.

**Comment by hebertialmeida** [comment]
@hunght I have tested it, and it is working nice, just one think that I have noticed is the Now playing info is not updating, there is a function for this `updateNowPlayingInfo()`. Is it for you to implement this?

**Pull request merged**